### PR TITLE
Make `RealmScreen` more similar to the Electron app.

### LINKED ISSUE #1709: Change server URL input label/hint on RealmScreen.
We should make the wording here similar to the electron app.
Something like:
"Register or log in to a Zulip organization to get started"

"URL of Zulip organization"
[input box with placeholder of: acme.zulipchat.com or chat.acme.com], this should accept with or without http, with or without /register or /login at the end.

Or [create a new organization on zulipchat.com] (link goes to zulipchat.com/create_realm)

https://chat.zulip.org/#narrow/stream/new.20members/subject/server.20URL/near/429550

### PR DESCRIPTION
**Pull request by skunkmb**
Add a header above the URL input and add a link to create a new organization underneath. Also make some changes to placeholders and button text. See discussion here: https://goo.gl/QCZgLY.

Also, add style props to `WebLink` to help design the new `RealmScreen`.

**Screenshot**:

new `RealmScreen` screenshot

Fix #1709.

### CODE REVIEW

**Comment by skunkmb** [comment]
Tests are failing, but it seems like that is caused by the `Flow` error that has been causing all of the recent commits and PRs to fail since b70f7bff9ecb630cbdd2329fd8981452d67f821e, not just this one. It seems like that might be fixed by #1704 soon.

**Comment by skunkmb** [comment]
Update: tests are passing now.

File: src/start/RealmScreen.js [hunk 1]
```diff
@@ -85,10 +101,16 @@ class RealmScreen extends PureComponent<Props, State> {
         {error && <ErrorMsg error={error} />}
         <ZulipButton
           style={styles.smallMarginTop}
-          text="Enter"
+          text="+ Add"
```
Comment (jainkuniya) [inline src/start/RealmScreen.js]: `Enter` will be more appropriate here.

File: src/start/RealmScreen.js [hunk 2]
```diff
@@ -68,13 +80,17 @@ class RealmScreen extends PureComponent<Props, State> {
...
+          style={[styles.heading2, customStyles.heading]}
+        />
+        <Label text="URL of Zulip organization" />
         <Input
           style={styles.smallMarginTop}
           autoFocus
           autoCorrect={false}
           autoCapitalize="none"
-          placeholder="Server URL"
+          placeholder="your-organization.zulipchat.com"
```
Comment (jainkuniya) [inline src/start/RealmScreen.js]: Also please add it `zulip-mobile/src/i18n/translations/messages_en.json`

File: src/start/RealmScreen.js [hunk 3]
```diff
@@ -68,13 +80,17 @@ class RealmScreen extends PureComponent<Props, State> {
 
     return (
       <Screen title="Welcome" padding scrollView>
-        <Label text="Your server URL" />
+        <Label
+          text="Register or log in to a Zulip organization to get started."
```
Comment (jainkuniya) [inline src/start/RealmScreen.js]: Also please add it `zulip-mobile/src/i18n/translations/messages_en.json`

File: src/start/RealmScreen.js [hunk 4]
```diff
@@ -85,10 +101,16 @@ class RealmScreen extends PureComponent<Props, State> {
...
         <ZulipButton
           style={styles.smallMarginTop}
-          text="Enter"
+          text="+ Add"
           progress={progress}
           onPress={this.tryRealm}
         />
+        <WebLink
+          style={customStyles.newOrgLink}
+          labelStyle={customStyles.newOrgLinkLabel}
```
Comment (jainkuniya) [inline src/start/RealmScreen.js]: Creating a `wrapper` for adding marginTop & centre will be better, instead of passing styles object.

Comment (borisyankov) [reply src/start/RealmScreen.js]: Not sure if I agree. This looks ok.

File: src/start/RealmScreen.js (hunk 3 above)
Comment (skunkmb) [reply src/start/RealmScreen.js]: @jainkuniya OK, I added it to `zulip-mobile/src/i18n/translations/messages_en.json`. Thanks!

File: src/start/RealmScreen.js (hunk 2 above)
Comment (skunkmb) [reply src/start/RealmScreen.js]: @jainkuniya OK, I added it to `zulip-mobile/src/i18n/translations/messages_en.json`. Thanks!

File: src/start/RealmScreen.js (hunk 1 above)
Comment (skunkmb) [reply src/start/RealmScreen.js]: @jainkuniya OK, I changed it back to `Enter`. Thanks!

**Comment by skunkmb** [comment]
@borisyankov Is there anything else I should change, or is this ready to be merged?

File: src/start/RealmScreen.js (hunk 4 above)
Comment (skunkmb) [reply src/start/RealmScreen.js]: @jainkuniya @borisyankov OK, for now I'll keep this as-is. Please let me know if there is anything else I should change. Thanks!

**Comment by jainkuniya** [comment]
Now we have reworked on `RealmScreen`, so we can close this.

Thanks! @skunkmb for the contribution, great work 🎉  :)

**Pull request closed without merging**